docs/reST/ref/mask.rst

```diff
@@ -158,26 +161,41 @@ Starting from pygame 1.9.5 masks with width or height 0 are supported.
       | :sl:`Sets all bits to 0`
       | :sg:`clear() -> None`
 
-      Sets all bits in a Mask to 0.
+      Sets all bits in the mask to 0.
+
+      :returns: ``None``
+      :rtype: None
 
       .. ## Mask.clear ##
 
    .. method:: invert
 
-      | :sl:`Flips the bits in a Mask`
+      | :sl:`Flips all the bits`
       | :sg:`invert() -> None`
```

Comment:
I wonder if it's bad that the return type/value is repeated?

Reply:
Good point, the type should be `NoneType`. I've updated the documentation to fix this.